test/Colors.test.js

```diff
@@ -0,0 +1,45 @@
+const COLORS = require("../src/Colors");
+const assert = require("assert");
+
+test("Colors Test", () => {
+	const color = COLORS.randomColor();
+	color.forEach(elem => {
+		assert.equal(typeof elem, "number");
+	});
+});
+
+test("should return the same color three", () => {
+	const exclusionList = [];
+	Object.keys(COLORS).forEach(key => {
+		if (key !== "AQUA" && key !== "randomColor") {
```

Comment:
Checking `randomColor` here makes me think we should have the actual color values in a separate key in `Colors`. Maybe `colors.colors`? What do you think?

Reply:
I went ahead and pulled it apart, so now you can bring in either just the randomColor function or the object of colors, or just a specific color if you wanted